content/blog/2025/kai-release-announcement/index.md

```diff
@@ -0,0 +1,71 @@
+---
+author: SavithaRaghunathan, JonathanRecinos
+date: 2025-04-01
+description: >-
+    After months of ideas, development, and feedback from early users and contributors, we're excited to share the first official release of Konveyor AI 0.1.0
+featured: true
+image: header.png
+tags:
+    - Konveyor
+    - Kai
+    - GenAI
+    - AppModernization
+title: "Konveyor AI 0.1.0 is here!"
+---
+
+**Author**: [Savitha Raghunathan](https://github.com/savitharaghunathan), [Jonathan Recinos](https://github.com/JonathanR19)
+
+After months of ideas, development, and feedback from early users and contributors, we're excited to share the first official release of **[Konveyor AI 0.1.0](https://github.com/konveyor/editor-extensions/releases/tag/v0.1.0)**.
+
+This release is a significant step forward for the Konveyor project, and for anyone working through the complex journey of application modernization. It marks the beginning of **developer-focused, AI-assisted modernization** built with community input and open collaboration.
+
+## What is Konveyor AI?
+
+**Konveyor AI (Kai)** provides real-time insights to help developers and migrators identify and refactor deprecated or outdated code. As a developer modernizes an application with Kai, they'll see detailed issues and their location, solutions with the reasoning for it, and code suggestions that can be automatically implemented.
+
+Built on top of the Konveyor ecosystem, Kai brings:
+
+- **Static code analysis**  
+  Identifies potential issues when adopting new technologies (e.g., modern Java frameworks, containers).
+
+- **Changelog history from source repositories**  
+  Maintains a record of changes during migration.
+
+- **Issue tracking**  
+  Keeps a history of resolved modernization challenges.
+
+- **Enhanced migration intelligence**  
+  Includes 2,400+ predefined rules for various migration paths, with support for user-defined custom rules.
+
+- **VS Code extension**  
+  Provides modernization insights and suggested changes directly inside the IDE.
+
+- **Model-agnostic AI**  
+  Kai does not lock users into a specific LLM, providing flexibility for AI-driven workflows.
+
+- **Agentic AI for intelligent responses**  
+  Delivers more meaningful, sanitized suggestions, validating Maven compilations and dependency resolutions.
+
+- **Effort estimation for dependencies**  
```

Comment:
I would consider removing this block.
I don't think it reflects what we are doing

Reply:
removed it